whats_fresh/whats_fresh_api/templates/vendor.html

```diff
@@ -48,15 +48,40 @@ <h4 id='error_tag' class='light warning'></h4>
         {{ vendor_form.story }}
         <p class=" field_text">In Port:</p>
         {{ vendor_form.status }}
-        <h3>Location</h3>
-        <p class="field_text">Street Address*:</p>
-        {{ vendor_form.street }}
-        <p class="field_text">City*:</p>
-        {{ vendor_form.city }}
-        <p class="field_text">State*:</p>
-        {{ vendor_form.state }}
-        <p class="field_text">Zipcode*:</p>
-        {{ vendor_form.zip }}
+        <h3>Location</h3><br>
+
+        <script type="text/javascript" src='http://maps.google.com/maps/api/js?sensor=false&libraries=places'></script>
+
+    <!-- Real Input Address Field -->
+
+        <div>
+        <input type="text" id="map-address" onkeypress="removerEnter(event)" name="location" required="true"  disabled="disabled" />
+        </div>
+
+    <!-- Dummy Input address field-->
+
+        <div>
+        <input type="text" id="map-address-fake" onkeypress="removerEnter(event)" required="true" disabled="disabled" />
+        </div>
+
+        <label><input type="checkbox" id="address-checkbox" {% if vendor_form.instance.pk %} checked="checked" {% else %} checked="unchecked" {% endif %}/>Lock Address Field</label>
```

Comment:
And then we still have to make sure that the text field is initialized properly by default so we can type in it.